CMakeLists.txt

```diff
@@ -20,49 +20,38 @@ else()
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -pedantic")
 endif()
 
-# Define macro for setting cxx11 as standard for building a target.
-macro(use_cxx11 target)
```

Comment:
Maybe we could keep using this macro, but it would not have if-else, it would just have the piece of code under `else`, so we don't have to repeat it for each target.
And then we call this macro on each target where we want to use cxx11, as it was so far (which is all of them except hello world).